patterns/atoms/media/video/video.twig

```diff
@@ -0,0 +1,4 @@
+<video width="{{video.width}}" height="{{video.height}}" controls="{{video.controls}}" class="{{video.class}}">
```

Comment:
may be it makes sense to have all attributes defined in `{{video.attributes}}` and have a loop here to render all available attributes for video tag?